certified-connectors/DomainToolsIrisEnrich/apiProperties.json

```diff
@@ -1,56 +1,82 @@
 {
-  "properties": {
-    "connectionParameters": {
-      "username": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "API username",
-          "description": "The API username for this api",
-          "tooltip": "Provide the API username",
-          "constraints": {
-            "tabIndex": 2,
-            "clearText": true,
-            "required": "true"
-          }
-        }
-      },
-      "password": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "API key",
-          "description": "The API key for this api",
-          "tooltip": "Provide the API key",
-          "constraints": {
-            "tabIndex": 3,
-            "clearText": false,
-            "required": "true"
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#fffff9",
-    "capabilities": [],
-    "policyTemplateInstances": [
-      {
-        "templateId": "setqueryparameter",
-        "title": "Set Username",
-        "parameters": {
-          "x-ms-apimTemplateParameter.name": "api_username",
-          "x-ms-apimTemplateParameter.value": "@connectionParameters('username')",
-          "x-ms-apimTemplateParameter.existsAction": "override"
-        }
-      },
-      {
-        "templateId": "setqueryparameter",
-        "title": "Set API Key",
-        "parameters": {
-          "x-ms-apimTemplateParameter.name": "api_key",
-          "x-ms-apimTemplateParameter.value": "@connectionParameters('password')",
-          "x-ms-apimTemplateParameter.existsAction": "override"
-        }
-      }
-    ],
-    "publisher": "DomainTools, LLC",
-    "stackOwner": "DomainTools Integrations"
-  }
+    "properties": {

```

Comment:
@RamboV please check the indentation in this file, it looks like contents of the file moved a bit.